src/main/java/seedu/address/logic/parser/ParserUtil.java

```diff
@@ -36,16 +37,59 @@
 
     public static final String MESSAGE_INVALID_INDEX = "Index is not a non-zero unsigned integer.";
 
+    public static final String MESSAGE_INDEX_OUT_OF_BOUNDS = "The index provided is invalid";
```

Comment:
Very minor nitpick, remember to add in a full stop here 😅

Reply:
Okay thanks for the keen observation!